python/lsst/ip/isr/isrTask.py

```diff
@@ -318,23 +320,7 @@ class IsrTaskConfig(pexConfig.Config):
 
 
 class IsrTask(pipeBase.CmdLineTask):
-    """!
-    @anchor IsrTask_
-
-    @brief Apply common instrument signature correction algorithms to a raw frame.
-
-    @section ip_isr_isr_Contents Contents
-
-     - @ref ip_isr_isr_Purpose
-     - @ref ip_isr_isr_Initialize
-     - @ref ip_isr_isr_IO
-     - @ref ip_isr_isr_Config
-     - @ref ip_isr_isr_Debug
-
-
-    @section ip_isr_isr_Purpose Description
-
-    The process for correcting imaging data is very similar from camera to camera.
+    """The process for correcting imaging data is very similar from camera to camera.
```

Comment:
I would have the first line here be Apply common instrument signature correction algorithms to a raw frame.